utils.go

```diff
@@ -1,42 +1,34 @@
 package musttag
 
 import (
+	"bytes"
+	"encoding/json"
 	"fmt"
 	"os/exec"
 	"strings"
 )
 
-// mainModule returns the directory and the set of packages of the main module.
-func mainModule() (dir string, packages map[string]struct{}, _ error) {
-	// https://pkg.go.dev/cmd/go#hdr-Package_lists_and_patterns
-	// > When using modules, "all" expands to all packages in the main module
-	// > and their dependencies, including dependencies needed by tests of any of those.
+type modInfo struct {
+	Path      string `json:"Path"`
+	Dir       string `json:"Dir"`
```

Comment:
We can now get the module's dir from here (previously, it was used at `musttag.go:167` to print relative paths)

Reply:
you don't need to use relative path inside report (at least for golangci-lint)

Reply:
I was talking about reporting the position of the function call (`fnPos token.Position` in the `report` function), not the position of the struct type (which is trimmed by `golangci-lint` indeed). AFAIR, without using `filepath.Rel`, `fnPos` will be the full path

Reply:
Coming back to our previous conversation, I think you were right about just reporting the function call instead of the struct type 😄 That would be much simpler

Reply:
yes, you have chosen the most "difficult" way :wink:

Just for context, our previous conversation was here: https://github.com/golangci/golangci-lint/pull/3624

Reply:
I just tested it again, and it looks like `golangci-lint` prettifies all the paths in the report. Cool, you're right then!

Reply:
I'm gonna implement a simpler version reporting just the function call position in a separate PR later